packages/design/tailwind/css/components.css

```diff
@@ -943,3 +943,17 @@ input[type='file' i] {
 }
 
 /* End of Breadcrumbs */
+
+/* Table */
+
+.gi-table {
+  @apply gi-w-full gi-border-spacing-0 gi-border-collapse gi-text-md;
+}
+
+.gi-table-th,
+.gi-table-td {
+  @apply gi-text-left gi-align-top gi-pl-1 gi-pr-5 gi-py-2.5 gi-border-b-[gray] gi-border-b gi-border-solid gi-align-middle;
+  @apply gi-max-w-[25ch] gi-overflow-hidden gi-text-ellipsis gi-whitespace-nowrap;
```

Comment:
is there a reason why we have 2 `@apply` here ?

Reply:
Just for readability, the second apply is for applying ellipsis, max characters etc. I can revert it if you guys feel uncomfortable with this 😄 

@pregno @ramonmulia

Reply:
yes, let's keep only one thanks